eng/pipelines/dotnet-monitor-publish.yml

```diff
@@ -0,0 +1,55 @@
+trigger: none
+pr: none
+
+parameters:
+- name: IsTestRun
+  type: boolean
+  default: true
+- name: IsDryRun
+  type: boolean
+  default: true
+
+variables:
+- template: /eng/common/templates-official/variables/pool-providers.yml@self
+- name: _TeamName
+  value: DotNetCore
+  readonly: true
+- group: Release-Pipeline
+- name: IsDryRun
+  value: ${{ parameters.IsDryRun }}
+  readonly: true
+- name: IsTestRun
+  value: ${{ parameters.IsTestRun }}
+  readonly: true
+
+resources:
+  pipelines:
+  - pipeline: dotnet-monitor_build
+    project: internal
+    source: 'dotnet\dotnet-monitor\dotnet-dotnet-monitor'
```

Comment:
This probably should just be `dotnet-dotnet-monitor`. See the `dotnet-monitor-release` pipeline: https://github.com/dotnet/dotnet-monitor/blob/main/eng/pipelines/dotnet-monitor-release.yml#L27

Reply:
I'm not sure why partial specification is preferred? I have no idea what the resolution process here is.